media_driver/agnostic/common/cm/cm_mem_sse2_impl.cpp

```diff
@@ -31,6 +31,12 @@
 
 #include <mmintrin.h>
 
+// GCC>=11 gives a warning when dividing sizeof(array) with sizeof(another)type)
+// in case division was mistyped and meant to calculate array size.
+// That's not out case here, so extra parenthesis for the divisor suppresses
+// the warning.
+#define DQWORD_PER_PREFETCH(P) ( sizeof(P)/(sizeof(DQWORD)) )
```

Comment:
Few typos: 
s/another)type/another type/
s/not out case/not the case/

And the last clause would read better with a verb: "As that's not the case here, use extra (divisor) parenthesis to suppress the warning."

LGTM otherwise.